### DISCUSSION TITLE
Twilio "create message" request causing Vercel deployment to fail silently

### DISCUSSION THREAD

**Discussion by nakamolto** [vercel/community#3253] [Help]
### Summary

Hey all - 

I'm trying to build an app that sends a text message using Twilio when a user enters a text and submits a form. IT WORKS GREAT WHEN RUNNING LOCALLY, including using "vercel dev". However, when I deploy it to Vercel.com, it fails silently when the code attempts to execute a Twilio "create message" request. Specifically:

const twilioClient = require('twilio')(twilioAccountSID, twilioAuthToken);
const message = await twilioClient.messages
    .create({
       body: response,
       from: twilioPhoneNumber,
       to: toPhoneNumber
     })

results in the code ending execution. If I use a try catch the catch doesn't execute. Console logs after this code block do not execute.

I've verified with console logs that the twilioClient object is what it should be and that my environment variables are set correctly. Strangely, the run didn't run out of memory, and it finished executing in 1.7 seconds (far less than 10 so it isn't a time out). 

Does anyone have any idea what I'm doing wrong?

### Steps to Reproduce

Run the following code as an API in a Vercel deployment, it will fail!

const twilioClient = require('twilio')(twilioAccountSID, twilioAuthToken);
const message = await twilioClient.messages
    .create({
       body: response,
       from: twilioPhoneNumber,
       to: toPhoneNumber
     })

**Comment by nakamolto** [comment]
Update. In my latest deployment, the code is executing intermittently. It seems to work at random, and then stop working for periods of time. At this point I'm pretty sure there is nothing wrong with my code and that there must be some sort of Vercel deployment issue. Anybody have any ideas? I've tried using Axios to replace the Twilioclient and I'm running into the same issue.

**Comment by JJwilkin** [comment]
I have encountered the same thing, in my case it sends the response object back to the client indicating the POST request was successful but the text message is never sent

**Comment by gamerxsocietydev** [comment]
Does anyone have a solution for this? I am running into the same issue. Would love guidance on a workaround for this if found. Thank you!

**Comment by zacharymichaelmoore** [comment]
Does vercel have a vpn blocking this and they are unable to parse it?

**Comment by mint-nguyen** [comment]
Running on the same issue. Does anyone have a solution yet?
